### ISSUE TITLE
figure out why PublishRelease fails on npm

### ISSUE THREAD

**Issue by Gankra** [axodotdev/oranda#577]
workflow: https://github.com/axodotdev/oranda/blob/main/.github/workflows/publish-release.yml

* passing run in cargodisttest: https://github.com/Gankra/axolotlsay/actions/runs/5557935085
* failing run in oranda: https://github.com/axodotdev/oranda/actions/runs/5754007165/job/15598920514

**Comment by Gankra** [comment]
One of the notable variables between these two is the the *kind* of token:

https://docs.npmjs.com/about-access-tokens

For cargodisttest I made a granular access token (completely arbitrarily, the ui prompted me to use it as the New Hotness).

For oranda I believe we made a legacy access token (because it can be non-expiring).

This particular section on legacy tokens seems concerning, although the error message is very cryptic if this is the problem:

> You can use these tokens to download packages, install packages, and update user and package settings. We recommend using them for interactive workflows such as a CLI. If 2FA is enabled on your account, publish tokens will require 2FA to execute sensitive operations on npm.


-------------------

Also note that we're running a slightly modified version of [the workflow recommended by Github's Docs](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages). The differences are that we aren't running from within the package we're publishing (we're publishing a premade tarball), and we don't run `npm ci` (I forget why I removed that, I think it didn't make sense with tarballs).

Note that `actions/setup-node` apparently creates this [.npmrc](https://docs.npmjs.com/cli/v7/configuring-npm/npmrc) file

```
//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
registry=https://registry.npmjs.org/
always-auth=true
```

I'm not sure if that formulation has issues with legacy tokens (or tarball publishes, but again cargodisttest worked).
